patches/chrome-browser-resources-settings-site_settings-category_default_setting.ts.patch

```diff
@@ -10,10 +10,11 @@ index 52dffb565b6557a65d07ce05cb5487018e78b88b..182a20ebb5e24d6d35b1443746b88bde
        case ContentSettingsTypes.BACKGROUND_SYNC:
        case ContentSettingsTypes.FEDERATED_IDENTITY_API:
        case ContentSettingsTypes.IMAGES:
-@@ -218,6 +219,7 @@ export class CategoryDefaultSettingElement extends
+@@ -218,6 +219,8 @@ export class CategoryDefaultSettingElement extends
        case ContentSettingsTypes.USB_DEVICES:
        case ContentSettingsTypes.VR:
        case ContentSettingsTypes.WINDOW_MANAGEMENT:
++      case ContentSettingsTypes.GOOGLE_SIGN_IN:
```

Comment:
this should not be added on a new line, it should go on the same line as the existing ones

Reply:
`ContentSettingsTypes.AUTOPLAY` is on its own line though above, is there a reason to crowd them on a single line?

Reply:
yes, patches should be a single line whenever possible because it minimizes the chance of conflicts. We do this everywhere